src/server/utils/getCashaySchema.js

```diff
@@ -3,4 +3,6 @@ require('babel-polyfill');
 const {transformSchema} = require('cashay');
 const graphql = require('graphql').graphql;
 const rootSchema = require('../graphql/rootSchema');
+const r = require('../database/rethinkDriver');
```

Comment:
we can comment out the `const r...` like we do the line below it. perhaps we should think of a way to use cashay-loader inside the webpack config itself?